src/languages/robotstxt.js

```diff
@@ -0,0 +1,44 @@
+/*
+Language: robots.txt
+Author: Thomas LÉVEIL <thomasleveil@gmail.com>
+Description: language definition for robots.txt files
+Category: config
+*/
+
+function(hljs) {
+  var HASH_COMMENT_MODE = hljs.COMMENT('(?:^|\\s+)#', '$');
+  return {
+    aliases: ['robotstxt', 'robots.txt'],
+    case_insensitive: true,
+    lexemes: '[a-z-]+',
+    keywords: {
+      section: 'user-agent',
+      built_in: 'allow disallow',
+      keyword: 'crawl-delay sitemap'
+    },
+    contains: [
+      HASH_COMMENT_MODE,
+      hljs.NUMBER_MODE,
+      {
+        className: 'regex', 
```

Comment:
Why not string?  Isn't * the only special character allowed and these aren't regexes?